test/selenium/util/TestUser.scala

```diff
@@ -0,0 +1,21 @@
+package selenium.util
+
+import java.time.Duration.ofDays
+
+import com.gu.identity.testing.usernames.TestUsernames
+
+class TestUser {
+
+  private val testUsers = TestUsernames(
+    com.gu.identity.testing.usernames.Encoder.withSecret(Config.testUsersSecret),
+    recency = ofDays(2)
+  )
+
+  private def addTestUserCookies(testUsername: String) = {
+    Driver.addCookie("ANALYTICS_OFF_KEY", "true")
```

Comment:
Yep, sorry, I may be guilty of copying this class directly out of membership-frontend... However, I agree that it would be useful to implement this behaviour in support-frontend. I'll create a separate card for this though, as it's beyond the scope of this PR.